index.html

```diff
@@ -58,6 +59,216 @@ <h1 class="identity_text">I'm Simba, I'm glad to see you!</h1>
+      <h2 class="project_title">Tonic</h2>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio3.png" class="project_img" alt="project_image">
+      <h2 class="project_title">Multi-Post Stories</h2>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+  </section>
+
+  <section class="aboutme_container">
+    <div class="aboutme_class">
+      <h2 class="identity_text">About Myself</h2>
+      <p class="description_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p class="connect_text">LET'S CONNECT</p>
```

Comment:
- [ ] Please LET'S CONNECT is a title. Kindly use the `<h1,2,..>` header instead of paragraph `<p>`. Open this[ LINK](https://www.w3docs.com/learn-html/semantic-elements-in-html5.html) to learn more about "Semantic HTML"